ocean_lib/ocean/mint_fake_ocean.py

```diff
@@ -16,14 +14,12 @@
 from ocean_lib.web3_internal.web3_provider import Web3Provider
 
 
-def mint_fake_OCEAN():
+def mint_fake_OCEAN(config):
```

Comment:
any changes to mint_fake_OCEAN must be reflected in the READMEs, specifically the marketplace flow README. Please change that readme in this same PR, and test it thoroughly.

Reply:
Good catch!  Thanks.  I'll do this and comment here when I'm done.

Reply:
I updated marketplace-flow.md and tested.  `mint_fake_OCEAN` executed successfully. Unfortunately, I ran into an error later in the workflow. 

However, I believe the error is not caused by my changes.  I verified this by repeating the workflow using the `main` branch, and the same error occurred.

Thus, I do not think there are any problems with the changes included in this PR. I created #345 to track the error I observed.